lx_anonymizer/ollama_service.py

```
@@ -0,0 +1,213 @@
+import json
+from logging import root
+from pathlib import Path
+import requests
+from .custom_logger import get_logger
+import subprocess
+import time
+import subprocess
```

Ruff (0.8.2)

Redefinition of unused `subprocess` from line 6

Remove definition: `subprocess`

(F811)